corehq/ex-submodules/casexml/apps/case/templates/case/partials/case_attachments.html

```diff
@@ -6,8 +6,6 @@
     {% if not forloop.first %}<br>{% endif %}
     <dt>{% trans "Case Property" %}</dt>
         <dd>{{ attach_key }}</dd>
-    <dt>{% trans "Source File" %}</dt>
-        <dd>{{ attachment.attachment_src }}</dd>
     <dt>{% trans "File Type" %}</dt>
         <dd>{{ attachment.server_mime }}</dd>
```

Comment:
Just seeing this now and unrelated but this should be `attachment.content_type`

Reply:
5745015c2a29e64860e48f90e46903a9b957746b